docker/publication-server-docker.conf

```diff
@@ -38,6 +38,7 @@ akka.http.server.request-timeout = ${?HTTP_REQUEST_TIMEOUT}
 
 # listen on ipv4 only
 server.address="0.0.0.0"
+server.address=${?SERVER_ADDRESS}
```

Comment:
Is this a strange way to default to `0.0.0.0` or does it bind to two addresses this way?

Reply:
It's the default